caikit/runtime/service_factory.py

```diff
@@ -238,7 +219,10 @@ def get_service_package(
                 # We need the message class that data model serializes to
                 setattr(client_module, dm_class.__name__, type(dm_class().to_proto()))
 
-            service_json = cls._create_service_json(task_rpc_list, package_name)
+            rpc_jsons = []
+            for rpc in task_rpc_list:
+                rpc_jsons.append(rpc.create_rpc_json(package_name))
```

Comment:
could probably replace with a list comprehension as well

Reply:
oh man, just re-reading this code and the logic for creating the `package_name` is gnarly lol.

Gotta think about that one more, maybe when we introduce `@task`s they should come with a package name?

Reply:
Now that you mention. I think this logic is redundant:
```
ai_domain_name = snake_to_upper_camel(lib.replace("caikit_", ""))
```
if our `lib` is sample_lib. This might have been more for our internal use case.

I think we need to revisit this when `task decorator` is introduced